<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>StyleSettingsWidget</class>
 <widget class="QWidget" name="StyleSettingsWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>448</width>
    <height>456</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QGroupBox" name="Header1">
     <property name="minimumSize">
      <size>
       <width>430</width>
       <height>100</height>
      </size>
     </property>
     <property name="title">
      <string>Text</string>
     </property>
     <widget class="QLabel" name="fontLabel">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>31</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Font:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>60</y>
        <width>41</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Color:</string>
      </property>
     </widget>
     <widget class="QFontComboBox" name="fontBox">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>20</y>
        <width>112</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QComboBox" name="sizeBox">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>60</y>
        <width>51</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="colorPicker">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>60</y>
        <width>32</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QComboBox" name="justify">
      <property name="geometry">
       <rect>
        <x>300</x>
        <y>60</y>
        <width>111</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>181</x>
        <y>60</y>
        <width>101</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Horizontal Alignment:</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="bold">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>20</y>
        <width>41</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Bold</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="italicize">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>20</y>
        <width>70</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Italicize</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="underline">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>20</y>
        <width>70</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Underline</string>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>60</y>
        <width>41</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Size:</string>
      </property>
     </widget>
     <zorder>label_2</zorder>
     <zorder>fontLabel</zorder>
     <zorder>fontBox</zorder>
     <zorder>sizeBox</zorder>
     <zorder>colorPicker</zorder>
     <zorder>justify</zorder>
     <zorder>label_3</zorder>
     <zorder>bold</zorder>
     <zorder>italicize</zorder>
     <zorder>underline</zorder>
     <zorder>label</zorder>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QFrame" name="frame">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>1</verstretch>
      </sizepolicy>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QLabel" name="prev">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>80</y>
        <width>151</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Test area!</string>
      </property>
     </widget>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QGroupBox" name="bod">
     <property name="minimumSize">
      <size>
       <width>430</width>
       <height>110</height>
      </size>
     </property>
     <property name="title">
      <string>Container</string>
     </property>
     <widget class="QPushButton" name="colorPicker_bg">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>30</y>
        <width>31</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QSpinBox" name="width">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>70</y>
        <width>43</width>
        <height>20</height>
       </rect>
      </property>
      <property name="maximum">
       <number>500</number>
      </property>
     </widget>
     <widget class="QSpinBox" name="height">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>70</y>
        <width>43</width>
        <height>20</height>
       </rect>
      </property>
      <property name="maximum">
       <number>500</number>
      </property>
     </widget>
     <widget class="QSpinBox" name="margin">
      <property name="geometry">
       <rect>
        <x>320</x>
        <y>70</y>
        <width>41</width>
        <height>20</height>
       </rect>
      </property>
      <property name="maximum">
       <number>200</number>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>8</x>
        <y>30</y>
        <width>31</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Color:</string>
      </property>
     </widget>
     <widget class="QComboBox" name="justify_2">
      <property name="geometry">
       <rect>
        <x>300</x>
        <y>30</y>
        <width>111</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>30</y>
        <width>91</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Vertical Alignment:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>11</x>
        <y>70</y>
        <width>31</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Width:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>70</y>
        <width>35</width>
        <height>13</height>
       </rect>
      </property>
      <property name="text">
       <string>Height:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_8">
      <property name="geometry">
       <rect>
        <x>280</x>
        <y>70</y>
        <width>36</width>
        <height>13</height>
       </rect>
      </property>
      <property name="text">
       <string>Margin:</string>
      </property>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
